sklearn/tests/test_docstring_parameters.py

```diff
@@ -190,7 +191,7 @@ def test_fit_docstring_attributes(name, Estimator):
     if Estimator.__name__ in IGNORED or Estimator.__name__.startswith('_'):
         pytest.skip("Estimator cannot be fit easily to test fit attributes")
 
-    est = Estimator()
+    est = _construct_instance(Estimator)
```

Comment:
`OneVsRestClassifier` needs a estimator to be constructed.

Reply:
Thanks :D